### DISCUSSION TITLE
`NITRO_PRESET=firebase` build only for `us-central1` ?

### DISCUSSION THREAD

**Discussion by monsat** [nuxt/nuxt#17957] [Questions] [answered]
I have checked 
https://github.com/nuxt/framework/blob/main/packages/nitro/src/runtime/entries/firebase.ts

The output will be deployed to `us-central1`

I'd like to deploy to `asia-northeast1`

> https://firebase.google.com/docs/functions/manage-functions#modify-region

Is it possible to add `.region('asia-northeast1')` or any region config?

**Comment by danielroe** [comment, marked as answer]
@monsat I've created an issue to follow (https://github.com/nuxt/framework/issues/3126). Until then, it is possible for you to create [your own custom Nitro preset](https://v3.nuxtjs.org/docs/deployment/presets/custom) to do what you need.

**Reply by monsat** [reply]
Thanks 😄 
I'll create my own preset.
